activesupport/lib/active_support/railtie.rb

```diff
@@ -13,6 +13,13 @@ class Railtie < Rails::Railtie # :nodoc:
       end
     end
 
+    initializer "active_support.halt_callback_chains_on_return_false", after: :load_config_initializers do |app|
```

Comment:
@rafaelfranca this sounds like a bug^W possibly unintended behaviour I've run across before. After you define an `initializer(before/after: ..)`, subsequent `initializer` definitions with no special ordering requirements (like the three below) will be inserted immediately after that one, instead of "falling" to their more natural position.

When I hit it, I just worked around it without investigating whether it was deliberate. Maybe it's not? Certainly my default expectation is that adding a definition like this should be a simple insertion, and have no effect on the order of the others.